test/gerrit_tests/test_gerrit.py

```diff
@@ -388,6 +397,145 @@ def test_request_reviews_no_repos(self,
         mock_format_response.assert_not_called()
         self.assertEqual(None, response)
 
+    @patch(PATH + 'GerritService.format_response')
+    @patch(PATH + 'GerritService._filter_invited', return_value=[])
+    @patch(PATH + 'GerritService._call_api')
+    @patch(PATH + 'GerritService.get_response', return_value=True)
+    @patch(PATH + 'requests')
+    def test_request_reviews_with_reviewers_config(self,
+                                                   mocked_requests,
+                                                   mocked_get_response,
+                                                   mocked_call_api,
+                                                   mocked_filter_invited,
+                                                   mocked_format_response):
+        """Ensure _filter_invited is called when reviewers config is
+        provided."""
+
+        mocked_call_api.return_value = [
+            {'id': 'change_id'},
+        ]
+
+        reviewers_config = {
+            'excluded': []
+        }
+
+        service = GerritService()
+
+        service.request_reviews(
+            'host', 'repo', reviewers_config=reviewers_config
+        )
+
+        assert mocked_requests.session.called
+        assert mocked_get_response.called
+        assert mocked_filter_invited.called
+        called_args, _ = mocked_filter_invited.call_args
```

Comment:
can this section be refactored to `mocked_filter_invited.called_once_with(...)`?

Reply:
Yep. I changed the asserts for consistency with the other tests.